chat-plugins/trivia.js

```diff
@@ -864,9 +882,9 @@ const commands = {
 		if (room.id !== 'trivia') return this.errorReply("This command can only be used in Trivia.");
 		if (!room.game) return this.errorReply("There is no game of trivia in progress.");
 		if (room.game.gameid !== 'trivia') {
-			return this.errorReply("There is already a game of " + room.game.title + " in progress.");
+			return this.errorReply(`There is already a game of ${room.game.title} in progress.`);
 		}
-
+		if (room.game.phase !== SIGNUP_PHASE && !room.game.canLateJoin) return this.errorReply("This game does not allow latejoins.");
```

Comment:
`Trivia#addPlayer` should be checking this so commands don't get mixed up in how `Trivia` manages its state. The command can throw any string returned as an error reply and send the default signup message in the other case instead